--- a/scripts/mysqld_multi.sh	2019-11-07 13:52:22.000000000 +0800
+++ b/scripts/mysqld_multi.sh	2019-12-26 18:21:15.693848747 +0800
@@ -30,7 +30,7 @@
 $opt_help          = 0;
 $opt_log           = undef();
 $opt_mysqladmin    = "@bindir@/mysqladmin";
-$opt_mysqld        = "@sbindir@/mysqld";
+$opt_mysqld        = "@bindir@/mysqld";
 $opt_no_log        = 0;
 $opt_password      = undef();
 $opt_tcp_ip        = 0;
--- a/scripts/mysql_install_db.sh	2019-11-07 13:52:22.000000000 +0800
+++ b/scripts/mysql_install_db.sh	2019-12-26 18:45:28.593340878 +0800
@@ -301,7 +301,7 @@
   builddir="."
   print_defaults="./extra/my_print_defaults"
 else
-  print_defaults="@bindir@/my_print_defaults"
+  print_defaults="/usr/bin/my_print_defaults"
 fi
 
 if test ! -x "$print_defaults"
@@ -372,14 +372,14 @@
   plugindir="$basedir/@INSTALL_PLUGINDIR@"
   pamtooldir=$plugindir
 else
-  basedir="@prefix@"
-  bindir="@bindir@"
+  basedir="/usr"
+  bindir="/usr/bin"
   resolveip="$bindir/resolveip"
-  mysqld="@sbindir@/mysqld"
-  srcpkgdatadir="@pkgdatadir@"
-  buildpkgdatadir="@pkgdatadir@"
-  plugindir="@pkgplugindir@"
-  pamtooldir="@pkgplugindir@"
+  mysqld="/usr/bin/mysqld"
+  srcpkgdatadir="/usr/share/mariadb"
+  buildpkgdatadir="/usr/share/mariadb"
+  plugindir="/usr/lib/mariadb/plugin"
+  pamtooldir="/usr/lib/mariadb/plugin"
 fi
 
 # Set up paths to SQL scripts required for bootstrap
